dask_ml/preprocessing/_encoders.py

```diff
@@ -137,13 +140,13 @@ def __init__(
         super(OneHotEncoder, self).__init__(**signature)
 
     @classmethod
-    def _get_param_names(cls):
+    def _get_param_names(cls: Any):
```

Comment:
Does this need to be typed? Or is it like `self` and doesn't need to be?

A return type would be helpful though.